docs/team/brandonywl.md

```diff
@@ -16,21 +16,31 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 ### Features implemented
 - Set up and designed the architecture used for Event scheduling in the timetable
     - Handled `Event`, `RecurringEvent`, `Timetable`, `Reminder` and other relevant `Command` subclasses for Event scheduling functionality.
+    - Event functionality was designed with scalability in mind. Immutability was largely enforced to ensure consistency of Events even across re-occurrences. Implemented Single-Responsibility-Principle, Tell-Don't-Ask and Liskov Substitution Principle in Designs.
+    - Designed to reduce circular dependencies to reduce couplin.
 - Assisted in development of two common `Parser` methods
 
 ### Code contributed
-
+[Reposense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other&tabOpen=true&tabType=authorship&tabAuthor=brandonywl&tabRepo=AY2021S1-CS2113-T13-1%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other)
 ### Contributions to User Guide
 1. Maintain portions on commands related to Events.
     - `add-e`
     - `edit-e`
     - `list-e`
     - `delete-e`
+    - `remind-e`
 
 ### Contributions to Developer Guide
 1. AddEventCommand Sequence Diagram
 1. RemindCommand Sequence Diagram
-1. Timetable and Event Class Diagram
+1. Timetable Class Diagram
+1. Event and RecurringEvent Class Diagram
+
+### Review Contributions:
+1. [Code Review](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/47)
+1. [Code Cleanup](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/66)
+1. [Assist in Standardization](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/71)
+1. [Provided Assistance in Event Functionality Saving & Edge Case](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/122)
 
 ### Community
 (https://nus-cs2113-ay2021s1.github.io/addressbook-level3/team/johndoe.html)
```

Comment:
If no community involvement can remove. But IIRC, the code reviews should come under this section
```suggestion
(https://nus-cs2113-ay2021s1.github.io/addressbook-level3/team/johndoe.html)
```